app/models/container_node.rb

```diff
@@ -85,4 +86,20 @@ def evaluate_alert(_alert_id, _event)
     # and these should automaticaly be translated to alerts.
     true
   end
+
+  supports :external_logging_support do
+    unless ext_management_system.respond_to?(:external_logging_route_name)
+      unsupported_reason_add(:external_logging_support, _('This provider type does not support external_logging_support'))
+    end
+  end
+
+  def external_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def external_logging_path
+    hostname = (ipaddress || name || "").split('.').first
+    query = "bool:(filter:(or:!((term:(hostname:#{hostname})),(term:(hostname:#{name})))))"
+    "/#/discover?_g=()&_a=(columns:!(_source),filters:!((meta:(disabled:!f,index:'.operations.*',key:hostname,negate:!f),#{query})),index:'.operations.*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'*')),sort:!(time,desc))"
```

Comment:
Can you factor out the `.operations.*` index name as a variable?  It appears in two places in the string.  Also, I tried to use this URL with a few adjustments for names in my Kibana instance, but I get an error.  Are you sure this is right?

Additionally, could we add some fields by default so that it is not just the `_source` field, which is large blob?  Perhaps adding `hostname`, `level`, `pod_name`, and `message` would be sufficient initially.